pkg/types/clusterconfig/errors.go

```diff
@@ -335,17 +352,31 @@ func ErrorThroughputNotSupported(volumeType VolumeType) error {
 	})
 }
 
-func ErrorIOPSTooLarge(iops, maxIOPS int64) error {
+func ErrorIOPSTooSmall(volumeType VolumeType, iops, minIOPS int64) error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrIOPSTooSmall,
+		Message: fmt.Sprintf("for %s volume type, %s (%d) cannot be smaller than %d", volumeType, InstanceVolumeIOPSKey, iops, minIOPS),
+	})
+}
+
+func ErrorIOPSTooLarge(volumeType VolumeType, iops, maxIOPS int64) error {
 	return errors.WithStack(&errors.Error{
 		Kind:    ErrIOPSTooLarge,
-		Message: fmt.Sprintf("%s (%d) cannot be larger than %d", InstanceVolumeIOPSKey, iops, maxIOPS),
+		Message: fmt.Sprintf("for %s volume type, %s (%d) cannot be larger than %d", volumeType, InstanceVolumeIOPSKey, iops, maxIOPS),
+	})
+}
+
+func ErrorIOPSToVolumeSizeRatio(volumeType VolumeType, ratio, iops int64, volumeSize int64) error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrIOPSToVolumeSizeRatio,
+		Message: fmt.Sprintf("for %s volume type, %s (%d) cannot be more than %d times larger than %s (%d); increase `%s` or decrease `%s` in your cluster configuration file", volumeType, InstanceVolumeIOPSKey, iops, ratio, InstanceVolumeSizeKey, volumeSize, InstanceVolumeSizeKey, InstanceVolumeIOPSKey),
 	})
 }
 
-func ErrorIOPSTooLargeDueToVolumeSize(iops int64, volumeSize int64) error {
+func ErrorIOPSToThroughputRatio(volumeType VolumeType, ratio, throughput int64, iops int64) error {
 	return errors.WithStack(&errors.Error{
-		Kind:    ErrIOPSTooLargeDueToVolumeSize,
-		Message: fmt.Sprintf("%s (%d) cannot be more than 50 times larger than %s (%d); increase `%s` or decrease `%s` in your cluster configuration file", InstanceVolumeIOPSKey, iops, InstanceVolumeSizeKey, volumeSize, InstanceVolumeSizeKey, InstanceVolumeIOPSKey),
+		Kind:    ErrIOPSToThroughputRatio,
+		Message: fmt.Sprintf("for %s volume type, %s (%d) cannot be less than %d times smaller than %s (%d); increase `%s` or decrease `%s` in your cluster configuration file", volumeType, InstanceVolumeIOPSKey, iops, ratio, InstanceVolumeThroughputKey, throughput, InstanceVolumeThroughputKey, InstanceVolumeIOPSKey),
```

Comment:
for gp3 volume type, iops (3000) must be at least 4 times larger than throughput (1000); decrease throughput or increase iops in your cluster configuration file